lib/cli/lib/helpers.js

```diff
@@ -206,23 +206,33 @@ export function installDependencies(npmOptions, dependencies) {
   const { skipInstall } = npmOptions;
 
   if (skipInstall) {
-    const spawnCommand = 'pkg-add-deps';
-
-    const installArgs = dependencies;
+    const { packageJson } = npmOptions;
 
     if (npmOptions.installAsDevDependencies) {
-      installArgs.push('-D');
+      packageJson.devDependencies = {
+        ...packageJson.devDependencies,
+        ...dependencies.reduce((acc, i) => {
+          const c = i.lastIndexOf('@');
```

Comment:
@ndelangen what kind of code is this???

Reply:
- `@storybook/addons@^5.3.0`
- find the last occurring '@'
- split the string on that char
- first part is the package name, second part is the version
- if version is undefined make it '*'

- add this to package.json, either in dependencies or devDependencies

Reply:
Yeah, and it's duplicated 5 lines below?

Reply:
yes. This solved a major blocker and I didn't mind some duplication as long as we were able to actually test the CLI again.

I'm sorry about some code-duplication. I'll get around to extracting it into a function some day.

Reply:
🤦‍♂

Reply:
https://github.com/storybookjs/storybook/pull/9004

Please get your code reviewed. I appreciate that it's useful to be able to self-merge, especially in the face of deployment issues where you need to iterate. In that case, please feel free to `@` me (or anybody else on the project) to do a review after the fact. Also, go back and clean up after a fire has been put out, or at the very least please be open to doing that when somebody calls you out on it.

Flaunting contribution guidelines and littering the codebase with temporary hacks--especially ones that take just a few seconds to fix--doesn't set a good example for anybody involved in the project.